eng/CodeGeneration.targets

```diff
@@ -5,18 +5,23 @@
     <_AutoRestCoreVersion>3.0.6257</_AutoRestCoreVersion>
     <_AutoRestCSharpVersion>https://github.com/Azure/autorest.csharp/releases/download/3.0.0-dev.20200325.2/autorest-csharp-v3-3.0.0-dev.20200325.2.tgz</_AutoRestCSharpVersion>
     <_SupportsCodeGeneration Condition="'$(IsClientLibrary)' == 'true'">true</_SupportsCodeGeneration>
-    <AutoRestConfiguration Condition="'$(AutoRestConfiguration)' == ''">autorest.md</AutoRestConfiguration>
-
+    <_DefaultInputName Condition="Exists('$(MSBuildProjectDirectory)/autorest.md')">$(MSBuildProjectDirectory)/autorest.md</_DefaultInputName>
+    <AutorestInput Condition="'$(AutorestInput)' == ''">$(_DefaultInputName)</AutorestInput>
+    <AutorestConfiguration Condition="'$(AutorestConfiguration)' == '' AND '$(AutorestInput)' != '$(_DefaultInputName)'">$(_DefaultInputName)</AutorestConfiguration>
+    <AutorestAdditionalParameters></AutorestAdditionalParameters>
```

Comment:
Nice!  Can you add a quick comment with details showing how to enable interactive mode?  I think that'll be helpful to point folks at when things go sideways.

Reply:
Haha, never used it. I'll figure it out and add a comment.

Reply:
Add `--interactive` in there and it should just light up.  It'll open a diagram, but the interesting bit is clicking "open all outputs in VS code" if you want to see exactly how your transforms have mangled the swagger files.